_conferences/mbltdev-2018.md

```diff
@@ -0,0 +1,11 @@
+---
+name: "MBLT DEV 2018"
```

Comment:
Can we drop the year from the name? The date is displayed on the website already.

Reply:
I think it is the part of the name, but I just asked the organizer and they said it's OK to name it without the year. So it's gone now.